db/migrate/20200827183737_create_identities.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+# Migration to add Identity table.
+class CreateIdentities < ActiveRecord::Migration[6.0]
+  def change
+    create_table :identities do |t|
+      t.references :user, null: false, foreign_key: true
+      t.integer :provider_uid, null: false, limit: 8
```

Comment:
This column should probably be a `string`. We should be able to handle if future providers use uuids instead of integers.

Reply:
updated provider_uid column to `string`